pkg/cache/v3/delta.go

```diff
@@ -31,10 +31,13 @@ func respondDelta(request *DeltaRequest, value chan DeltaResponse, state stream.
 	}
 
 	// Only send a response if there were changes
-	if len(resp.Resources) > 0 || len(resp.RemovedResources) > 0 {
+	if len(resp.Resources) > 0 || len(resp.RemovedResources) > 0 ||
+		// we want to respond immediatly for the first wildcard request in a stream, even if the response is empty
+		// otherwise, envoy won't complete initialization
```

Comment:
Does this only apply for CDS/LDS since this is targeted for wildcard mode?

Reply:
Yes, I added the check for  wildcard based on this comment https://github.com/envoyproxy/go-control-plane/issues/465#issuecomment-885708998

Reply:
Okay great. Can we address the nit I had with the formatting?

Reply:
Sure, done!